### PR TITLE
kuberetes --> kubernetes

### PR DESCRIPTION
**Pull request by tbarnes-us**
Address Issue #2258 

I searched for all occurrences using `find . -name "*.*" -type f | xargs grep -li kuberetes` and ignored doc from older previous releases.

### CODE REVIEW

**Review by lennyphan — APPROVED** [review]
LGTM

File: documentation/staging/content/samples/simple/azure-kubernetes-service/domain-on-pv.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ secret/domain1-weblogic-credentials labeled
 The secret domain1-weblogic-credentials has been successfully created in the default namespace.
 ```
 
-We will use the `kubernetes/samples/scripts/create-kuberetes-secrets/create-docker-credentials-secret.sh` script to create the Docker credentials as a Kubernetes secret. Please run:
+We will use the `kubernetes/samples/scripts/create-kubernetes-secrets/create-docker-credentials-secret.sh` script to create the Docker credentials as a Kubernetes secret. Please run:
```
Comment (alai8) [inline documentation/staging/content/samples/simple/azure-kubernetes-service/domain-on-pv.md:108]: do we need to rename that directory too? (perhaps it is already renamed as part of this PR but I couldn't tell for sure).

Comment (rjeberhard) [reply documentation/staging/content/samples/simple/azure-kubernetes-service/domain-on-pv.md:108]: It does look like we need to use `git mv` to fix the name of the directory too.

Comment (tbarnes-us) [reply documentation/staging/content/samples/simple/azure-kubernetes-service/domain-on-pv.md:108]: Good catch! Sorry - I forgot.

**Review by alai8 — APPROVED** [review]

**Pull request merged**